### ISSUE TITLE
Implement compilation for AtomicFetchOp for POWER

### ISSUE THREAD

**Issue by hernanponcedeleon** [hernanponcedeleon/Dat3M#57]

**Comment by hernanponcedeleon** [comment]
Merged into address-simplify branch.
Closing this issue depends on #60.

**Comment by hernanponcedeleon** [comment]
Merged into master by a600260830801abe4cc9180040a1a20ee45a1672
